techniques/fileDistribution/manageKeyValueFile/1.0/manage-key-value-file.st

```diff
@@ -0,0 +1,45 @@
+#####################################################################################
+# Copyright 2016 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+bundle agent manage_key_value_file {
+  vars:
+      # Define a space variable, as form trim the space
+      "space" string => " ";
```

Comment:
Probably in ncf_lib, it could be a kind of an extension of https://docs.cfengine.com/docs/3.7/reference-special-variables-const.html.